README.md

```diff
@@ -7,29 +7,34 @@ Node.js tools for Visual Studio 2017 is developed and managed here.
 
 
 Check out our documentation on [the NTVS wiki](https://github.com/Microsoft/nodejstools/wiki) 
-and our (old) [feature overview](https://channel9.msdn.com/events/Visual-Studio/Connect-event-2015/801) video on Channel 9.
+<!--and our (old) [feature overview](https://channel9.msdn.com/events/Visual-Studio/Connect-event-2015/801) video on Channel 9.
+-->
 
 Feel free to file issues or ask questions on our [issue tracker](http://github.com/Microsoft/nodejstools/issues), 
 and we welcome code contributions - see 
 [Contributing](https://github.com/Microsoft/nodejstools/blob/master/.github/CONTRIBUTING.md) for information. 
 
-## Visual Studio 2017 installation
+## Visual Studio 2019 installation
 
 The Node.js development workload is available as part of Visual Studio 2017 Community, Professional 
 and Enterprise. 
 To install, run the [normal VS installer](https://visualstudio.com/vs/downloads?wt.mc_id=github_microsoft_com) 
 and select the **Node.js development workload**.
 
-For the latest update to the Node.js tools install [Visual Studio Preview](https://www.visualstudio.com/vs/preview/).
+If you already have Visual Studio and only need to download the Node.js Tools, you can fine them [here](https://visualstudio.microsoft.com/downloads/?wt_mc_id=github_microsoft_com)
 
-## Visual Studio 2015 and earlier
 
-Prior to Visual Studio 2017, the Node.js tools were released as a standalone extension. We are no longer actively developing these versions, 
-but if you are unable to upgrade to Visual Studio 2017 yet, you can use these older installers to get the Node.js tools.
+## Visual Studio 2017 installation
 
-[NTVS 1.3.1](https://github.com/Microsoft/nodejstools/releases/tag/v1.3.1) is the latest available version for Visual Studio 2015.
+The Node.js development workload is available as part of Visual Studio 2017 Community, Professional 
```

Comment:
nit: Seems like repetitive text. We might want to change the wording.

Reply:
any suggestions on wording? is it that we name all the versions that you dont like?

Reply:
Maybe: "For Visual Studio 2017 Community, Professional and Enterprise, head to run the [2017 VS installer](https://visualstudio.microsoft.com/vs/older-downloads/) and choose the **Node.js development workload**. Please note you will need a Visual Studio Subscription to access this."

Does it looks better?

Reply:
Maybe:

"For Visual Studio 2017 Community, Professional and Enterprise, run the 2017 VS installer and choose the Node.js development workload. Please note you will need a Visual Studio Subscription to access this"

Reply:
Sounds good to me.